.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+
+jobs:
+  tests:
+    docker:
+      - image: circleci/python:3.7-browsers
```

Comment:
With the browsers image tag, we get browsers (to run selenium tests for example), but also the JDK.